backend/instancer/api/account.py

```diff
@@ -336,6 +336,30 @@ def login() -> ResponseReturnValue:
     }
 
 
+@blueprint.route("/logout", methods=["POST"])
+def logout() -> ResponseReturnValue:
+    """Logout the current user
+
+    Requires body param login_token"""
+    try:
+        token = request.form["login_token"]
+    except KeyError:
+        return {"status": "missing_login_token", "msg": "Missing login token"}, 401
+    try:
+        account = LoginToken.decode(token)
+    except ValueError:
+        return {"status": "invalid_login_token", "msg": "Invalid login token"}, 401
+
+    res = authentication.del_session(account.team_id)
+    if not res:
+        return {"status": "logout_failed", "msg": "Logout failed"}, 500
```

Comment:
Maybe something like "invalid token" or similar? It should only be server side error if rclient like throws an error or something, returning false just means the token is either invalid or was already invalidated/expired

Reply:
What I am trying to get at it here if rclient does not return 1 as the number of entries deleted. I will leave the status but I will update the message to invalid token for improved error messages.